content/yaml-code-signing/signing-android.md

```diff
@@ -48,15 +48,12 @@ This example shows how to set up code signing using Gradle.
   }
   ...
 ```
-2. Add your keystore and keystore details in the [`environment`](../getting-started/yaml#environment) section of the configuration file. [Encrypt](../building/encrypting/#encrypting-sensitive-data) your keystore file, keystore password (if keystore is password-protected), key alias and key alias password (if key alias is password-protected) and set the encrypted values to the following environment variables. Note that encrypting files requires you to base64 encode them locally and decode them during the build.
+2. Add your keystore and keystore details in the `Environment variables` section of the Codemagic UI. [`Secure`](../building/encrypting/#encrypting-sensitive-data) your keystore file, keystore password (if keystore is password-protected), key alias and key alias password (if key alias is password-protected) and set the values to the following environment variables. Note that securing files requires you to [`base64 encode`](https://docs.codemagic.io/variables/environment-variable-groups/#global-variables-and-secrets) them locally and decode them during the build.
```

Comment:
We normally use bold to highlight UI elements, so it should be **Environment variables** and **Secure**. Also note that the link for Secure points to a file we no longer have, the right link would be this: ../variables/environment-variable-groups/.
Also we should use relative links everywhere in docs and not full links. 
The base64 encode link currently do not point to the right section either, it should be this: ../variables/environment-variable-groups/#storing-sensitive-valuesfiles. So actually it looks like both links should take to the same section.

Reply:
I think the whole section should be expanded a bit. The variable names are still important, right? The way I would present the instructions is sth like this:
- Add your environment variables in Codemagic UI (either in application or team variables)
- The names of the variables and what to add there
- A note about encoding binary files and decoding them in yaml
- A note about checking **Secure**
- A note about saving them all to the same group so they can be important to yaml at once